Server/Routes/routes.js

```diff
@@ -55,8 +56,13 @@ module.exports = function (app, express) {
 		//res.send('Arrived at endpoint: ' + req.url);
 		UserMethods.updateContacts(req.params.user_id, req.body.contacts, res);
 	});
+/* ================ CONTACT ENDPOINTS ================= */
+	app.get('/userpath/:user_id/:trip_id', function (req, res) {
```

Comment:
you don't need the anonymous callback wrapping Trip.read.  You can pass `Trip.read` as the callback and it should work.